app/com/arpnetworking/kairos/client/models/Sampling.java

```diff
@@ -69,6 +70,28 @@ public Builder setPeriod(final Period value) {
             return this;
         }
 
+        /**
+         * Sets the count of units that the Sampling covers.
+         *
+         * @param value the count of units
+         * @return this {@link Builder}
+         */
+        public Builder setValue(final int value) {
```

Comment:
This should be an `Integer` and marked as `@Nullable` based on the constraints below. The conversion from `Integer` to `int` should be in the constructor of the class being built.